src/js/core/methods/GetAccountInfo.js

```diff
@@ -97,13 +96,18 @@ export default class GetAccountInfo extends AbstractMethod {
                 // device will be used in Discovery
                 willUseDevice = true;
             }
-            batch.coinInfo = coinInfo;
 
             // set firmware range
             this.firmwareRange = getFirmwareRange(this.name, coinInfo, this.firmwareRange);
+
+            return {
+                ...batch,
+                address_n,
+                coinInfo,
+            };
         });
 
-        this.params = payload.bundle;
+        // this.params = payload.bundle;
```

Comment:
? aha, this.params is already set and validated from above.

Reply:
removed